[2021-10-14 17:23:58,522] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: file_upload.csv_load manual__2021-10-14T11:48:45.369809+00:00 [queued]>
[2021-10-14 17:23:58,535] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: file_upload.csv_load manual__2021-10-14T11:48:45.369809+00:00 [queued]>
[2021-10-14 17:23:58,536] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-14 17:23:58,536] {taskinstance.py:1242} INFO - Starting attempt 1 of 2
[2021-10-14 17:23:58,536] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-14 17:23:58,553] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): csv_load> on 2021-10-14 11:48:45.369809+00:00
[2021-10-14 17:23:58,557] {standard_task_runner.py:52} INFO - Started process 17367 to run task
[2021-10-14 17:23:58,560] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'file_upload', 'csv_load', 'manual__2021-10-14T11:48:45.369809+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/file_loadin.py', '--cfg-path', '/tmp/tmpg1busyd9', '--error-file', '/tmp/tmp5xyu29id']
[2021-10-14 17:23:58,564] {standard_task_runner.py:77} INFO - Job 64: Subtask csv_load
[2021-10-14 17:23:58,597] {logging_mixin.py:109} INFO - Running <TaskInstance: file_upload.csv_load manual__2021-10-14T11:48:45.369809+00:00 [running]> on host user-Latitude-3410
[2021-10-14 17:23:58,637] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=file_upload
AIRFLOW_CTX_TASK_ID=csv_load
AIRFLOW_CTX_EXECUTION_DATE=2021-10-14T11:48:45.369809+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-14T11:48:45.369809+00:00
[2021-10-14 17:23:58,653] {python.py:152} INFO - Done. Returned value was:                      Name  ... Difference between Dose 2 to today(in Days)
0          Sushant Shinde  ...                                         NaN
1          Sushant Shinde  ...                                         NaN
2                   Zeya   ...                                         NaN
3                   rahul  ...                                         NaN
4                   gopal  ...                                         NaN
5                    raj   ...                                         NaN
6                 divesh   ...                                         NaN
7                     abc  ...                                         NaN
8                     zxc  ...                                         NaN
9                     lkj  ...                                         NaN
10         sushant shinde  ...                                         NaN
11         Sushant Shinde  ...                                         NaN
12         Suvarna Shinde  ...                                         NaN
13  Sushant Dilip  Shinde  ...                                        34.0
14                Ridhima  ...                                         NaN
15                 Saloni  ...                                         NaN
16         shubham shinde  ...                                         NaN
17                   Zeya  ...                                         NaN
18         Shubham Shinde  ...                                         NaN

[19 rows x 15 columns]
[2021-10-14 17:23:58,715] {xcom.py:333} ERROR - Could not serialize the XCom value into JSON. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your airflow config.
[2021-10-14 17:23:58,716] {taskinstance.py:1686} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1324, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1443, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1502, in _execute_task
    self.xcom_push(key=XCOM_RETURN_KEY, value=result)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 2237, in xcom_push
    session=session,
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/utils/session.py", line 67, in wrapper
    return func(*args, **kwargs)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/models/xcom.py", line 99, in set
    value = XCom.serialize_value(value)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/models/xcom.py", line 330, in serialize_value
    return json.dumps(value).encode('UTF-8')
  File "/usr/lib/python3.6/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python3.6/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.6/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python3.6/json/encoder.py", line 180, in default
    o.__class__.__name__)
TypeError: Object of type 'DataFrame' is not JSON serializable
[2021-10-14 17:23:58,727] {taskinstance.py:1280} INFO - Marking task as UP_FOR_RETRY. dag_id=file_upload, task_id=csv_load, execution_date=20211014T114845, start_date=20211014T115358, end_date=20211014T115358
[2021-10-14 17:23:58,740] {standard_task_runner.py:91} ERROR - Failed to execute job 64 for task csv_load
Traceback (most recent call last):
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1324, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1443, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1502, in _execute_task
    self.xcom_push(key=XCOM_RETURN_KEY, value=result)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 2237, in xcom_push
    session=session,
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/utils/session.py", line 67, in wrapper
    return func(*args, **kwargs)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/models/xcom.py", line 99, in set
    value = XCom.serialize_value(value)
  File "/home/user/workspace/airflow/airflow_env/lib/python3.6/site-packages/airflow/models/xcom.py", line 330, in serialize_value
    return json.dumps(value).encode('UTF-8')
  File "/usr/lib/python3.6/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python3.6/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.6/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python3.6/json/encoder.py", line 180, in default
    o.__class__.__name__)
TypeError: Object of type 'DataFrame' is not JSON serializable
[2021-10-14 17:23:58,774] {local_task_job.py:154} INFO - Task exited with return code 1
[2021-10-14 17:23:58,802] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
